packages/ibm-products-web-components/src/components/side-panel/side-panel.ts

```diff
@@ -0,0 +1,980 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2023, 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { LitElement, html } from 'lit';
+import {
+  property,
+  query,
+  queryAssignedElements,
+  state,
+} from 'lit/decorators.js';
+import { prefix, carbonPrefix } from '../../globals/settings';
+import HostListener from '../../globals/decorators/host-listener';
+import HostListenerMixin from '../../globals/mixins/host-listener';
+import { SIDE_PANEL_SIZE, SIDE_PANEL_PLACEMENT } from './defs';
+import styles from './side-panel.scss?lit';
+import { selectorTabbable } from '../../globals/settings';
+import { carbonElement as customElement } from '../../globals/decorators/carbon-element';
```

Comment:
```suggestion
import HostListener from '@carbon/web-components/es/globals/decorators/host-listener.js';
import HostListenerMixin from '@carbon/web-components/es/globals/decorators/host-listener.js';
import { SIDE_PANEL_SIZE, SIDE_PANEL_PLACEMENT } from './defs';
import styles from './side-panel.scss?lit';
import { selectorTabbable } from '../../globals/settings';
import { carbonElement as customElement } from '@carbon/web-components/es/globals/decorators/carbon-element.js';
```

Can import these directly from `@carbon/web-components`.